<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="output" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>321</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Q</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>F, MHz</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rsh, kOhm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_turn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_Q">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_freq">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>0.270000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.270000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_Rsh">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>100.000000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_turn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>N turns</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_n_turns">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>2000</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spin_turn</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_turn</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_turn</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_turn</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
